policy/predicate/file.go

```diff
@@ -37,13 +38,25 @@ func (pred *ChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (boo
 		return false, "", errors.Wrap(err, "failed to parse paths")
 	}
 
+	ignorePaths, err := pathsToRegexps(pred.IgnorePaths)
+	if err != nil {
+		return false, "", errors.Wrap(err, "failed to parse ignore paths")
+	}
+
 	files, err := prctx.ChangedFiles()
 	if err != nil {
 		return false, "", errors.Wrap(err, "failed to list changed files")
 	}
 
+	var cleanedFilenames []string
```

Comment:
Maybe `filteredFiles` or `relevanatFiles`? "Cleaned" makes me thing the names have been changed in some way.